src/server.rs

```diff
@@ -0,0 +1,294 @@
+use std::env;
+use fresh_eyes::{
+    extract_pr_details, get_pull_request_reviews, Branch as LibBranch,
```

Comment:
I can't see where we're using this function `get_pull_request_reviews`
Lets remove it

Reply:
I removed it already